libseccomp/src/filter_context.rs

```diff
@@ -487,6 +578,21 @@ impl ScmpFilterContext {
     ///
     /// If this function is called with an invalid filter, an issue is encountered
     /// getting the current state, or the libseccomp API level is less than 4, an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// let mut ctx = ScmpFilterContext::new_filter(ScmpAction::Allow)?;
+    /// if check_api(4, ScmpVersion::from((2, 5, 0))).unwrap() {
+    ///     ctx.set_ctl_ssb(true)?;
+    ///     assert!(ctx.get_ctl_ssb()?);
+    /// } else {
+    ///     assert!(ctx.set_ctl_ssb(true).is_err());
+    ///     assert!(ctx.get_ctl_ssb().is_err());
```

Comment:
`true` is the default IIRC

```suggestion
    ///     ctx.set_ctl_ssb(false)?;
    ///     assert!(!ctx.get_ctl_ssb()?);
    /// } else {
    ///     assert!(ctx.set_ctl_ssb(false).is_err());
    ///     assert!(ctx.get_ctl_ssb().is_err());
```

Reply:
Added the fix in [commit](https://github.com/libseccomp-rs/libseccomp-rs/commit/2e07e17f07f92e2b6f7766c3c2d67495dc762efe)